sb_vision/vision.py

```diff
@@ -76,11 +76,12 @@ def process_image(self, img: Image.Image) -> List[Token]:
         """
         img = self.threshold_image(img)
 
-        distance_model = self.camera.distance_model
+        camera_model = self.camera.camera_model
+        detections = list(self.apriltag_detector.detect_tags(img))
```

Comment:
Why are you casting this to a list, to then just iterate over it? I think keeping it as-is inside the list comprehension is much nicer

Reply:
Good catch, this was because I iterated over it twice at one point when experimenting, fixed in 523971493427503dd65aa6be8f967d4d25513b0d